google-cloud-contact_center_insights-v1/.repo-metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+    "api_id": "contactcenterinsights.googleapis.com",
+    "client_documentation": "https://googleapis.dev/ruby/google-cloud-contact_center_insights-v1/latest",
+    "distribution_name": "google-cloud-contact_center_insights-v1",
+    "language": "ruby",
+    "name": "contactcenterinsights",
+    "name_pretty": "Contact Center AI Insights V1 API",
+    "product_documentation": "https://cloud.google.com/contact-center/insights/",
```

Comment:
This URL redirects to `https://cloud.google.com/solutions/contact-center`. However, `https://cloud.google.com/contact-center/insights/docs` currently works, so it might be a better choice.

Reply:
I'll get that updated upstream after the initial generation. Thanks!